### DISCUSSION TITLE
CD Cover Art Not Saved

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1728] [Fixed Issues]
@rern 
I added a CD to the library in 20240719, and although the cover art is found and displayed, clicking on the SD symbol does not save it.

**Comment by R101G** [comment]
@rern
Latest update seems to have removed any option to save the cover art.

Edit: Also, clicking on update thumbnails takes you to the addons menu.

**Reply by klassikmann** [reply]
> Edit: Also, clicking on update thumbnails takes you to the addons menu.

Sometimes, not always.

**Reply by rern** [reply]
Please try the UPDATE:
- Fix: thumbnail update more than once
- Fix: audio CD coverart
- Improve: audio CD in Playlist

**Reply by R101G** [reply]
@rern 
Sorry for the delay in replying. I still cannot save the cover art by clicking on the SD symbol. Manually pasting the image into the album folder allows thumbnail creation without problems.

**Reply by klassikmann** [reply]
> > Edit: Also, clicking on update thumbnails takes you to the addons menu.
> 
> Sometimes, not always.

That issue had been fixed with the UPDATE at Aug 25th.

**Comment by R101G** [comment]
@rern 
This bug is still there. I rarely buy a new CD, so I am not sure if it was fixed for a while or not.
